shared/reducers/search.js

```diff
@@ -93,9 +94,11 @@ export default function (state: State = initialState, action: SearchActions): St
     case Constants.removeUserFromGroup:
       if (!action.error) {
         const user = action.payload.user
+        const nextSelectedUsers = state.selectedUsers.filter(u => u !== user)
         return {
           ...state,
-          selectedUsers: state.selectedUsers.filter(u => u !== user),
+          showUserGroup: nextSelectedUsers.length === 0 ? false : state.showUserGroup,
```

Comment:
:bike: This is too performant. Please use add JQuery as a dependency and use a [DOM size](https://api.jquery.com/size/) method instead.